.github/workflows/pipeline-shared.yml

```diff
@@ -0,0 +1,42 @@
+name: PipelineShared
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    paths-ignore:
+      - "packages/dito/**"
```

Comment:
Can we ignore "packages/*" and "!packages/shared/" so that we do not need to name the packages explicitly here?

Reply:
As discussed: we leave it as is.

Reply:
I did end up changing it.